### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by dskvr** [deliciousbrains/wp-background-processing#36]
@A5hleyRich There are issues going back more than a year, do you need someone to take over maintenance of this project?

**Comment by khromov** [comment]
@dskvr Just go ahead and fork it or use one of the 40 existing forks as of me writing this. 😄

**Comment by lkraav** [comment]
Hmmm. I just discovered this and am wondering what the use case difference to https://github.com/Prospress/action-scheduler is. AS is incorporated into several commercial products with momentum, so unlikely to die any time soon.

**Comment by robskrob** [comment]
@lkraav thank you for sharing https://github.com/Prospress/action-scheduler -- indeed this project looks well maintained. However, it seems to be strictly for scheduling jobs in the future to run one time or scheduling recurring jobs on specific dates / times. I could be wrong, but I think `WP Background Processing` is just a background process that runs as soon as it can. Although the mechanics of [`bernard`](https://github.com/bernardphp/bernard) are very different from `WP Background Processing`, they seem to be similar in that they just execute jobs to be run offloaded in the background without a scheduling component to what they do. 

In short, I am not too sure if `Action Scheduler` is a perfect substitute for `WP Background Processing` seeing as `Action Scheduler` mainly handles scheduled jobs and not just a job that run's asynchronously to the web process (`apache`) which enqueues the job for asynchronous execution.

**Comment by robskrob** [comment]
Interesting, one of the maintainers of the project [claims](https://github.com/Prospress/action-scheduler/issues/250#issuecomment-470323315) it can kinda function like an asynchronous process without any scheduling concept associated to it.

**Comment by lkraav** [comment]
> Interesting, one of the maintainers of the project claims it can kinda function like an asynchronous process without any scheduling concept associated to it.

Yup, `time()` schedule sounds good to me.

**Comment by michaelschofield** [comment]
@lkraav @robskrob wow this is hella good timing. At work we finally got to a point where we need to start doing this kind of processing and as much as I trust DB I was concerned this project was dead. I think I'm going to give aciton-scheduler a try.

**Comment by anderly** [comment]
All, fwiw, the team behind action-scheduler, Prospress, was recently acquired by Automattic. It's already being used in WooCommerce with plans to use it more in the future. It's probably the best, most supported alternative currently out there.
